src/gtimelog/paths.py

```diff
@@ -25,14 +25,14 @@
             print("Failed: %s" % e)
 
 
-ui_dir = here
-
-UI_FILE = os.path.join(ui_dir, 'gtimelog.ui')
-PREFERENCES_UI_FILE = os.path.join(ui_dir, 'preferences.ui')
-ABOUT_DIALOG_UI_FILE = os.path.join(ui_dir, 'about.ui')
-SHORTCUTS_UI_FILE = os.path.join(ui_dir, 'shortcuts.ui')
-MENUS_UI_FILE = os.path.join(ui_dir, 'menus.ui')
-CSS_FILE = os.path.join(ui_dir, 'gtimelog.css')
+ui_dir = os.path.join(here, 'ui')
+views_dir = os.path.join(ui_dir, 'views')
+CSS_FILE = os.path.join(views_dir, 'gtimelog.css')
+UI_FILE = os.path.join(views_dir, 'gtimelog.ui')
+PREFERENCES_UI_FILE = os.path.join(views_dir, 'preferences.ui')
+ABOUT_DIALOG_UI_FILE = os.path.join(views_dir, 'about.ui')
+SHORTCUTS_UI_FILE = os.path.join(views_dir, 'shortcuts.ui')
+MENUS_UI_FILE = os.path.join(views_dir, 'menus.ui')
```

Comment:
I don't quite follow why ui/ needs further subdivision into ui/views/ and ... what else is in ui/?

Reply:
.ui and .py are two seperates concepts. One manage de logics, the other the representation. It's just concepts after all (well, the fondation of everything btw ^^). I flatten them, it's structure is less visual but act the same for sure.

Reply:
I'd assumed there would be more _code_ subpackages sibling to gtimnelog.ui.components.  When I later realized that you put the code subdirectory sibling to the data files directory, I forgot to go back and edit my comment.

Reply:
In my mind, .ui files are subservient to .py files and should be in a subdirectory, so gtimelog/ui/*.py and gtimelog/ui/data/*.ui or similar.

Normally I'd put them in a directory called 'ui', but it's already taken, and I don't want gtimelog/ui/ui/.